huey/storage.py

```diff
@@ -104,8 +104,17 @@ def flush_all(self):
 
 
 class RedisStorage(BaseStorage):
-    def __init__(self, name='huey', blocking=False, read_timeout=1,
-                 max_errors=1000, connection_pool=None, **connection_params):
+    def __init__(self, name='huey', blocking=False, read_timeout=1, max_errors=1000,
+            connection_pool=None, url=None, **connection_params):
+        if sum(bool(x) for x in (url, connection_pool, connection_params)) > 1:
```

Comment:
No :( For compatibility with Python 3, `filter()` would need to be wrapped by `list()`:

```python
if len(list(filter(None, (url, connection_pool, connection_params)))) > 1:
```

IMHO this is a readability "downgrade" in comparison to the original code. If `len()` is a must have for you I would suggest using `len([x for x in (url, ...) if x]) > 1`, which works on all Pythons >= 2.6.